clients/tests/accuracy_test_adhoc.cpp

```diff
@@ -39,15 +39,6 @@ std::vector<std::vector<size_t>> adhoc_sizes = {
     // Failure with build_CS_3D_BLOCK_RC
     {680, 128, 128},
 
-    // Large 1D primes that fall above the block threshold (length 262144).
```

Comment:
Why were these test cases removed?

Reply:
They were only accidentally added (per @eng-flavio-teixeira ) and they cause buffer assignment to explode with combinations, using up all your host memory (assuming the test actually decides to try building the plan).  Precompilation does not check if the problem will fit, so it hits this problem.

Limiting the explosion would be helpful but is a separate problem, and once that's addressed we can consider re-adding this case.  But it won't fit on any GPUs we have currently.

Reply:
The accidental part of this was not realizing that by adding the entry {196597, 25165813} to adhoc_sizes, it would run the 2D problem 196597 x 25165813. The intention **was to only test the 1D problems 196597 and 25165813**. 

Could we just add the entries separately {196597} and {25165813} to adhoc_sizes, so that it tests only the 1D cases?

Reply:
Ah ok, we'll need to fix that in develop too then.  I misunderstood you earlier.